### PR TITLE
Updated .gitignore for Vagrant environments

### PR DESCRIPTION
**Pull request by coderste**
Added the necessary folders and files to the `.gitignore` so contributors don't commit their Vagrant Environment

### CODE REVIEW

**Comment by lonnieezell** [comment]
Thanks!

**Comment by coderste** [comment]
No worries :D Really want to start contributing to CodeIgniter4 so if there's anything you need help on let me know and I'll try my best to help 😄

**Comment by lonnieezell** [comment]
Always will welcome help! :) There are some issues here on GitHub that are marked `help wanted` that haven't been claimed by anyone, yet. There's usually a couple of un-resolved issues from users laying around that you can help out with.

If you are feeling really brave there's SQLite and MSSQL drivers that have to be ported over. 

If you just want to start with smaller stuff, it's always helpful to go through the and verify accuracy of documentation against what the code will actually do, or writing tests for any place missing tests, etc. 

Basically - dive right in and start submitting PR's no matter how large or small. On anything other than small items, though, I'd prefer if you open a new issue (or comment on an existing one) and we can discuss your plan for you, just to make sure we're all on the same page.

**Comment by coderste** [comment]
Just a quick little change as well the URL when in the repo does not work as a link - I think this is because there is a '/' at the end of the URL if you remove the '/' then it becomes a link 😃
